gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/PartitionedRegionHelper.java

```diff
@@ -1071,13 +1071,39 @@ public static String escapePRPath(String prFullPath) {
     return escaped;
   }
 
-  public static final String TWO_SEPARATORS = LocalRegion.SEPARATOR
-      + LocalRegion.SEPARATOR;
-
   public static String unescapePRPath(String escapedPath) {
+    // below logic is broken if separators are mixed with underscores
```

Comment:
Can you please add a small comment of what this method is doing in normal case with normal pr region names.